.changeset/young-walls-prove.md

```diff
@@ -0,0 +1,10 @@
+---
+'@backstage/core-plugin-api': minor
+'@backstage/core-app-api': minor
+'@backstage/integration-react': minor
```

Comment:
not sure if this is necessary? the only thing changed is the api-report

Reply:
Let's split this up into multiple changesets. For core-plugin-api, the change is that the GitLab auth API must now also implement the OpenIdConnect interface. This is not considered a breaking change as our versioning policy allows breaking API producers.

For core-app-api we're added the `openid` scope, I don't think this should be a breaking change either?

You can indeed skip the changeset for integration-react

Reply:
Ah no adding the `openid` scope is totally a breaking change as it needs to be configured in the GitLab end too. Let's not ship that. Can we have the k8s plugin request the `openid` scope instead and document that for the GitLab integration? We won't need to core API changes with that

Reply:
OK, so now there's no more breaking changes to GitLab auth, but 456eaa8cf8378a7fad044abc20baa970aae0cd13 updates `OAuth2` to request the `openid` scope when getting ID tokens.